bin/test_wp_install.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+if [ $# -lt 3 ]; then
+  echo "usage: $0 <db-name> <db-user> <db-pass> [db-host] [wp-version]"
+  exit 1
+fi
+
+DB_NAME=$1
+DB_USER=$2
+DB_PASS=$3
+DB_HOST=${4-localhost}
+WP_VERSION=${5-latest}
+
+read -e -p "Where do you want to install wordpress? " WP_CORE_DIR
```

Comment:
Maybe install this test WordPress inside this plugin in a gitignored directory. That way you don't have to ask for a directory.